catalogue/webapp/components/Download/Download.js

```diff
@@ -23,65 +82,160 @@ const Download = ({
-            action: 'download small work image',
-            label: work.id,
-          }}
-          icon="download"
-          text="Download small (760px)"
-        />
-      </div>
-
-      {(iiifImageLocationCredit || iiifImageLocationLicenseId) && (
-        <div className={spacing({ s: 0 }, { margin: ['top'] })}>
-          {iiifImageLocationCredit && (
-            <p
-              className={classNames([
-                font({ s: 'HNL5', m: 'HNL4' }),
-                spacing({ s: 1 }, { margin: ['bottom'] }),
-              ])}
+        {useJavascriptControl ? (
+          <h2 className="inline">
+            <DownloadButton
+              className={classNames({
+                [font({ s: 'HNM4' })]: true,
+                'flex-inline': true,
+                'flex--v-center': true,
+              })}
+              aria-controls="downloadOptions"
+              aria-expanded={showDownloads}
+              rotateIcon={showDownloads}
+              onClick={() => {
+                setShowDownloads(!showDownloads);
+              }}
             >
-              Credit: {iiifImageLocationCredit}
-            </p>
-          )}
+              <span className="flex-inline flex--v-center">
+                <span
+                  className={classNames({
+                    [spacing({ s: 1 }, { margin: ['right'] })]: true,
+                  })}
+                >
+                  Download this image
+                </span>
+                <Icon name="chevron" />
+              </span>
+            </DownloadButton>
+          </h2>
+        ) : (
+          <h2
+            className={classNames({
+              [font({ s: 'WB6', m: 'WB5' })]: true,
+              'work-details-heading': true,
+            })}
+          >
+            Download this image
+          </h2>
+        )}
+        <DownloadOptions
+          id="downloadOptions"
+          className={classNames({
+            [font({ s: 'HNM5', m: 'HNM4' })]: true,
+            'enhanced-styles': useJavascriptControl,
+            show: showDownloads,
+          })}
+        >
+          <ul className="plain-list no-margin no-padding">
+            <li>
+              <a
+                tabIndex={showDownloads ? null : -1}
+                target="_blank"
+                rel="noopener noreferrer"
+                href={convertImageUri(iiifImageLocationUrl, 'full')}
+                onClick={() => {
+                  trackEvent({
+                    category: 'Button',
+                    action: 'download large work image',
+                    label: work.id,
+                  });
+                }}
+              >
+                <span className="flex-inline flex--v-center">
+                  <Icon name="download" />
+                  <span className="underline-on-hover">Download full size</span>
+                  <span
+                    className={classNames({
+                      'font-pewter': true,
+                      'font-HNM5-s': true,
```

Comment:
Appreciate that this isn't really the focus of this PR, but this should probably use the `font()` classes function rather than being jammed in manually.

Reply:
I personally like these things coming up as we refactor as we go.